Doc/library/ctypes.rst

```diff
@@ -147,6 +147,12 @@ functions can be accessed by indexing the dll object with the ordinal number::
 Calling functions
 ^^^^^^^^^^^^^^^^^
 
+.. deprecated-removed:: 3.6.2 3.7
+
+   Calling :mod:`ctypes` functions is deprecated. Calling a native function
+   with incorrect arguments is unsupported and can easily cause
+   information leakage or code execution vulnerabilities.
```

Comment:
Agreed. I'd get really specific here with something like "Calling `ctypes` functions with incorrect arguments, either number or type, is unsupported and may no longer raise a ValueError."

It might even make more sense to insert the note directly above the paragraph below that says "ctypes tries to protect you...". Or just remove that section and add a note saying "ctypes may raise a ValueError after calling the function if it detects that an invalid number of arguments were passed. This behavior is deprecated, should not be relied upon, and will be removed in Python 3.7"